### PR TITLE
DO-194: Added accordian functionality for family details

### PR DESCRIPTION
**Pull request by dhruvlahoti**

### CODE REVIEW

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateAddButton/ArrayTemplateAddButton.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Button from '../../../Button';
+
+const ArrayTemplateAddButton = ({ canAdd, onAddClick }) => {
+  if (canAdd) {
+    return (
+      <div className="array-item-add">
+        <p className="array-item-add__button">
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateAddButton/ArrayTemplateAddButton.jsx:10]: Button inside a para tag is an incorrect way to use it. Use span instead. Plus fix classNames as per the component.

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      );
+    }
+    return null;
+  };
+
+  const heading = `${element.children.props.formData.name}, ${element.children.props.formData.relationshipType}`;
+
+  const renderHeading = () => {
+    if (element.children.props.formData.name) {
+      return <p>{heading}<span><Icon icon={isEditClicked ? faMinus : faPlus} /></span></p>;
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:37]: Use Paragrapgh Typography component

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          {errors}
+        </div>
+      );
+    }
+    return null;
+  };
+
+  const renderTitle = () => {
+    if (title && title.length) {
+      return <p>{title}</p>;
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx:48]: Use Paragraph component

File: src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/familyDetailsSchema.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (mohitsaladiya) [inline src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/familyDetailsSchema.json:1]: Don't forget to add list this file on confluence.

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateAddButton/ArrayTemplateAddButton.jsx (hunk 1 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateAddButton/ArrayTemplateAddButton.jsx:10]: Its a replica of Array Template thats why using same calls name, will do this changes in refactoring

File: src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/familyDetailsSchema.json (hunk 4 above)
Comment (dhruvlahoti) [reply src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/familyDetailsSchema.json:1]: Will add after it gets merged.

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateAddButton/ArrayTemplateAddButton.jsx (hunk 1 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateAddButton/ArrayTemplateAddButton.jsx:10]: Adding a TODO

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx (hunk 3 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx]: Done

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx (hunk 2 above)
Comment (dhruvlahoti) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx]: Done

**Pull request merged**